rules-tests/Php84/Rector/Foreach_/ForeachToArrayAnyRector/Fixture/boolean_assignment_basic_usage.php.inc

```diff
@@ -68,4 +68,4 @@ class BasicUsage
     }
 }
 
-?>
\ No newline at end of file
+?>
```

Comment:
could you revert rename fixture files? it were fine as is and match with class name of fixture.

Reply:
sure, do you want me to create 2 separate folders for the fixtures of each pattern ? or do you prefer a different way to organize them ?

Reply:
just revert the rename fixtures, and add more fixture for return use case